tool/state-processor/src/vesting/mod.rs

```diff
@@ -4,15 +4,20 @@ use crate::*;
 impl Processor {
 	pub fn process_vesting(&mut self) -> &mut Self {
 		// Storage items.
-		// https://github.dev/paritytech/substrate/blob/19162e43be45817b44c7d48e50d03f074f60fbf4/frame/vesting/src/lib.rs#L188-L196
+		// https://github.dev/darwinia-network/substrate/blob/darwinia-v0.12.5/frame/vesting/src/lib.rs#L188
 		let mut vestings = Map::default();
 
+		// TODO: adjust decimals
+		// TODO: adjust block number
 		log::info!("take solo `Vesting::Vesting`");
-		self.solo_state.take_raw(&item_key(b"Vesting", b"Vesting"), &mut vestings, |key, from| {
-			replace_first_match(key, from, &item_key(b"AccountMigration", b"Vestings"))
-		});
+		self.solo_state.take_raw_map(
+			&item_key(b"Vesting", b"Vesting"),
+			&mut vestings,
+			|key, from| replace_first_match(key, from, &item_key(b"AccountMigration", b"Vestings")),
```

Comment:
Suggest renaming `from` to `prefix` this line.

Reply:
This will be changed later. There are some bugs in vesting processor.